product_docs/docs/edbcloud/beta/getting_started/01a_managing_quotas.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Managing service limits and constraints in Azure
+
+---
+
+Your Azure subscription must meet certain quota (service limit) requirements to work with EDB Cloud. By default, Azure sets a very low limit on the number of VMs, vCPUs (cores), and public IP addresses permitted for a particular region in your subscription.
+
+To enable EDB Cloud to perform its operations successfully, ensure that the following requirements are met.  
+
+ * Is there enough quota for vCPU and IP addresses in your region?
+ * Are there enough virtual machine SKUs available in your region?
+ * Are the necessary Azure resource providers registered to your subscription?
+
+For information about planning for quota requirements, see [Understanding quotas in Azure](../01_increasing_qotas_in_azure/01_understanding_qotas_in_azure). 
+
+Before you use EDB Cloud to deploy your clusters:
+
+1. [Check the quota requirements are met for your cluster](#check-qota-requirements).
+1. [Remediate quota issues in your Azure subscription](#remediate-qota-issues).
+
+## Check the quota requirements
+
+EDB provides a shell script called [`azure-preflight`](https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/azure-preflight.sh) to check your subscription automatically.  Although EDB recommends using the the `azure-preflight` script, you can also use the Azure CLI or the Azure portal to manually check the requirements.
+
+### Using the `azure-preflight` shell script
+
+Before proceeding, ensure that the following utilities are installed on the server where the script is run.
+ 
+ - [jq command-line JSON processor](https://stedolan.github.io/jq/) specific to your OS
+ - Azure CLI v2.26 or later
+
+1. From the Azure CLI, run the following command to download the script:  
+
+ ```
+ git clone https://github.com/EnterpriseDB/cloud-utilities/
+ ```
+
+1. Run the `azure-preflight.sh` script: 
+   ```
+  ./cloud-utilities/azure/azure-preflight.sh --location <azure_region> --pgtype <pg_type> --endpoint <end_point> [--ha] [--with-infra]
```

Comment:
We need to change the invocation here to align with the onboarding process. There are discussions happening on this point concurrent with this review; I'll report back with notes later this week.

Reply:
Sure, let me know of any updates.

Reply:
@ba-edb is there a ticket tracking this update?